### PR TITLE
[Admin] Maybe logs round station report

### PR DESCRIPTION
**Pull request by JamieD1**
Maybe adds logs to round start

#### Changelog

:cl:  
rscadd: Adds logs to round start
/:cl:

### CODE REVIEW

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -581,7 +581,7 @@
 
 	for (var/C in GLOB.admins)
 		to_chat(C, msg.Join())
-
+		log_admin("[msg.Join()]")
```
Comment (alexkar598) [inline code/game/gamemodes/game_mode.dm]: Why are you putting it in a string?

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/game_mode.dm (hunk 1 above)
Comment (JamieD1) [reply code/game/gamemodes/game_mode.dm]:
```suggestion
		log_admin([msg.Join()])
```

File: code/game/gamemodes/game_mode.dm [hunk 2]
```diff
@@ -581,7 +581,7 @@
 
 	for (var/C in GLOB.admins)
 		to_chat(C, msg.Join())
-
+		log_admin([msg.Join()])
```
Comment (alexkar598) [inline code/game/gamemodes/game_mode.dm]: EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

```suggestion
		log_admin([msg.Join())
```

**Review by alexkar598 — DISMISSED** [review]

File: code/game/gamemodes/game_mode.dm (hunk 2 above)
Comment (JamieD1) [reply code/game/gamemodes/game_mode.dm]: REEEEEEEEEEEEE

**Review by fluffe9911 — APPROVED** [review]
should be fine now

**Pull request merged**